### ISSUE TITLE
RF not being recommended for Moonkin running Theotar

### ISSUE THREAD

**Issue by Majcolorado** [Hekili/hekili#1181]
### Before You Begin

- [X] I confirm that I have downloaded the latest version of the addon.
- [X] I checked for an [existing, open ticket](https://github.com/Hekili/hekili/labels/recommendation) for this issue and was not able to find one.
- [X] I edited the title of this bug report (above) so that it describes the issue I am reporting.
- [X] I am reporting an issue with the default priority included with the specialization (imported or edited priorities are not supported).

### Describe the Issue

As described in the title.

### How to Reproduce

Pulls on Guardian of the First Ones

### Snapshot (Link)

https://pastebin.com/vHKG43Nd

### Contact Information

Rage#0949

**Comment by Hekili** [comment]
Thanks for the snapshot, however, this snapshot you provided is basically pre-pull -- you're not in Moonkin Form even.  Snapshots need to be taken at the time you think the CDs should've been recommended.  This would be sometime after you've initially opened (Wrath, Wrath, Starfire, possibly add dots, then you'd expect to see cooldown recommendations).

**Comment by Majcolorado** [comment]
I was able to recreate the issue on a target dummy, I'll attach two new Snapshots.

**Comment by Majcolorado** [comment]
https://pastebin.com/3FJgN3a2

https://pastebin.com/xF08Ycwj

**Comment by Hekili** [comment]
Are you macroing your IQD trinket and using it at a different time?  This is happening because it's trying to sync up with IQD but you already used it.

**Comment by Majcolorado** [comment]
Yes. That would explain why RF shows up briefly and then disappears once I hit my IQD/Starfire macro. Let me try again.

**Comment by Hekili** [comment]
I'm probably going to relax this IQD requirement, just because syncing is good but waiting a full CD to sync is bad.

**Comment by Majcolorado** [comment]
I macroed my IQD to RF and it's working fine now.

**Comment by Majcolorado** [comment]
That also solved the issue I posted in your Discord with Nadjia. It wasn't waiting for optimal Thrill Seeker buffs after all.
